easybuild/easyconfigs/h/Highway/Highway-1.0.4-zen4-fix-TruncateTo-bug.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/hwy/ops/x86_256-inl.h b/hwy/ops/x86_256-inl.h
```

Comment:
@trz42 Can you add a description on top of the patch to explain why this is required (fix failing test on AMD Genoa a.k.a. Zen4 systems), including a pointer to https://github.com/google/highway/issues/1913, and maybe also a bit of info on how you trimmed it down to just this minimal patch + add yourself as patch author.

Keep in mind that this implies updating the SHA256 checksum for the patch file in the easyconfig

Reply:
@trz42 Ping, please update the patch file with the requested info.
I need this asap :-)

Reply:
@boegel @akesandgren Added a description of the patch file. Let me know if you need additional information.

Note, I haven't tested the updated PR/easyconfig yet.

Reply:
Still builds for Zen4 on top of EESSI. See https://github.com/EESSI/software-layer/pull/627#issuecomment-2255653601